libpod/options.go

```diff
@@ -450,6 +467,19 @@ func WithShmDir(dir string) CtrCreateOption {
 	}
 }
 
+// WithContext sets the context to use.
+func WithContext(ctx context.Context) RuntimeOption {
```

Comment:
I feel like we ought to pass this through NewRuntime() as an argument instead of with an option